TeknoParrotUi.Common/Descriptions/CrimzonClover.json

```diff
@@ -1,7 +1,7 @@
 {
-  "platform": "Taito NESiCAxLive",
+  "platform": "Taito NESiCAxLive (TTX2)",
   "release_year": "2013",
-  "nvidia": "OK",
```

Comment:
why change this? it was already confirmed to be ok on nvidia.

Reply:
OK, re-issued.